### ISSUE TITLE
EventEmitter memory leak detected

### ISSUE THREAD

**Issue by azu** [azu/textlint#33]
When many rule is loaded, occur this warning
```
(node) warning: possible EventEmitter memory leak detected. 11 Str listeners added. Use emitter.setMaxListeners() to increase limit.
Trace
    at TextlintCore.addListener (events.js:239:17)
    at /Users/azu/.ghq/github.com/azu/JavaScript-Plugin-Architecture/node_modules/textlint/lib/textlint-core.js:31:16
    at Array.forEach (native)
    at addListenRule (/Users/azu/.ghq/github.com/azu/JavaScript-Plugin-Architecture/node_modules/textlint/lib/textlint-core.js:30:23)
    at /Users/azu/.ghq/github.com/azu/JavaScript-Plugin-Architecture/node_modules/textlint/lib/textlint-core.js:80:21
    at Array.forEach (native)
    at TextlintCore.setupRules (/Users/azu/.ghq/github.com/azu/JavaScript-Plugin-Architecture/node_modules/textlint/lib/textlint-core.js:65:32)
    at TextLintEngine.setupRules (/Users/azu/.ghq/github.com/azu/JavaScript-Plugin-Architecture/node_modules/textlint/lib/textlint-engine.js:81:22)
    at TextLintEngine.executeOnFiles (/Users/azu/.ghq/github.com/azu/JavaScript-Plugin-Architecture/node_modules/textlint/lib/textlint-engine.js:172:18)
    at Object.execute (/Users/azu/.ghq/github.com/azu/JavaScript-Plugin-Architecture/node_modules/textlint/lib/cli.js:75:70)
```

**Comment by azu** [comment]
Related issue https://github.com/eslint/eslint/issues/524
